spark/v3.3/spark/src/test/java/org/apache/iceberg/spark/sql/TestCreateTable.java

```diff
@@ -341,4 +343,42 @@ public void testDowngradeTableToFormatV1ThroughTablePropertyFails() {
         "Cannot downgrade v2 table to v1",
         () -> sql("ALTER TABLE %s SET TBLPROPERTIES ('format-version'='1')", tableName));
   }
+
+  @Test
+  public void testCatalogSpecificWarehouseLocation() throws Exception {
+    File warehouseLocation = temp.newFolder(catalogName);
+    Assert.assertTrue(warehouseLocation.delete());
```

Comment:
See https://iceberg.apache.org/contribute/#testing
For advice on asserts, we are using only AssertJ now

Reply:
Also why do we need to delete the location here?

Reply:
This does seem unnecessary.